bot/cogs/bot.py

```diff
@@ -8,7 +8,7 @@
 
 from bot.constants import (
     Channels, Guild, MODERATION_ROLES,
-    Roles, URLs,
+    URLs,
```

Comment:
Since this import is collectively 65 cars in length, it doesn't need to be multilined.